dapps/assets/src/components/digitaltwins/DigitalTwins.ts

```diff
@@ -48,6 +50,15 @@ export default class DigitalTwinsComponent extends mixins(EvanComponent) {
     required: true
   }) fetchMore: Function;
 
+  @Prop({
+    required: true
+  }) search: Function;
+
+  @Watch('searchTerm')
+    onSearchtermChanged(searchTerm: string, oldVal: string) {
```

Comment:
```suggestion
    onSearchTermChanged(searchTerm: string, oldVal: string) {
```
Fix function name accordingly